articles/prototyping-machine-learning-models-with-streamlit/index.md

```diff
@@ -0,0 +1,185 @@
+
+As Machine learning Engineers or Data scientist, you are used to building cool machine learning models but you are probably not familiar with building mobile or web apps, or the thought of building an application to serve your model is stressful because of the whole process of building either a web or mobile app so you just leave your model(s) on your notebook without using the weights. Streamlit is a platform that can help you prototype your model(s) in a few lines of code with less stress.
+#### Tabel of Content
+1. Introduction to streamlit
+2. Streamlit setup
+3. Building sentient analyzer
+4. Conclusion 
+
+#### Introduction to Streamlit
+Streamlit is an Open source python framework that enables developers to demo or prototype their ML models and a lot more on a web application effortlessly, you can simply turn your data science scripts into a website with few lines of code. It saves time, energy, and is also simple to learn, use, and fast too. Now we’ll dive straight into how to install and setup streamlit and then build a text summarizer web app.
+
+##### Streamlit Set up
+The installation is very simple, just run this on your command prompt or terminal
+
+```python
+Pip install streamlit
+Streamlit hello
+```
+To import it, use the simple
+
+```python
+import pandas as pd
+import streamlit as st
+import plotly.express as px
+```
+And to run your streamlit app simply use
+
+```python
+Streamlit run app.py
+```
+#### Building Sentiment Analyzer
+Now for this introductory tutorial, we will be building a sentiment analyzer model and prototype it with streamlit. The first step will be getting a Dataset, and the dataset we will be using can be found HERE, next let’s create a directory and create a virtual environment.
+
+```python
+$ mkdir textanalyzer
+$ cd textanalyzer
+```
+Now to create the Virtual environment use the code below
+
+```python
+$ python3.8 -m venv env
+$ source env/bin/activate
+```
+Now copy the dataset as it is in the folder and paste it into this textanalyzer directory, now create a file and save it as sentiment_analyzer, this will be where we will be writing our code to build the streamlit app. The next thing will be to import all the packages we will be using for this tutorial.
+
+```python
+import streamlit as st
+from sklearn.feature_extraction.text import CountVectorizer
+from sklearn.naive_bayes import BernoulliNB
+from sklearn.metrics import confusion_matrix
+import numpy as np
+import itertools
+import matplotlib.pyplot as plt
+```
+After that let’s add a header for our web app and a subheader and to do that we simply use the code below
+
+```python
+st.title("Sentiment Analyzer Based On Text Analysis ")
+st.subheader("Paras Patidar - MLAIT")
+st.write('\n\n')
+```
+The next step will be to write a function that will get all the data from our directory and bring them up,
```

Comment:
You need to insert an image of your directory structure. It is essential, and we need to ensure we include it whenever required.